library/src/main/java/uk/co/samuelwall/materialtaptargetprompt/extras/backgrounds/ImmersiveModeCompatPromptBackground.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2016-2018 Samuel Wall
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.co.samuelwall.materialtaptargetprompt.extras.backgrounds;
+
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.util.DisplayMetrics;
+import android.view.Display;
+import android.view.WindowManager;
+
+public class ImmersiveModeCompatPromptBackground extends FullscreenPromptBackground
```

Comment:
Could you add JavaDoc here describing the purpose for the class please

Reply:
Yes, of course. E.g. as here https://s.mail.ru/4DW8/XKexBcaBx ?